dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/conf/ConvolutionalLayerConfigurationBuilder.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2016 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.dolphin.dnn.conf;
+
+import edu.snu.dolphin.dnn.layerparam.initializer.ConvolutionalLayerParameterInitializer;
+import edu.snu.dolphin.dnn.layerparam.initializer.LayerParameterInitializer;
+import edu.snu.dolphin.dnn.layers.ConvolutionalLayer;
+import edu.snu.dolphin.dnn.layers.LayerBase;
+import edu.snu.dolphin.dnn.proto.NeuralNetworkProtos;
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.Tang;
+import org.apache.reef.util.Builder;
+
+/**
+ * Configuration builder for Convolutional layer.
+ *
+ * The configuration that this builder generates is used to create convolutional layer instance.
+ * The generated configuration needs to bind the implementation for matrix factory and
+ * the parameter for a layer input shape, to inject a layer instance.
+ */
+public final class ConvolutionalLayerConfigurationBuilder implements Builder<Configuration> {
+
+  public static ConvolutionalLayerConfigurationBuilder newConfigurationBuilder() {
+    return new ConvolutionalLayerConfigurationBuilder();
+  }
+
+  private int paddingHeight;
```

Comment:
Users do not have to specify `paddingHeight` when they use protocol buffer definition files. Shouldn't we set a default value for paddingHeight for consistency?
This is same for `paddingWidth`, `strideHeight`, and `strideWidth`.

If you agree with this, could you modify `PoolingLayerConfiguraitonBuilder`'s parameters that also have default values in the same way?